syntax = "proto3";

package network;

service Greeter {
    rpc SayHello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
    string message = 1;
}

message HelloReply {
    string message = 1;
}